recipes/ndarray_listener/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ndarray_listener" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "b921471c875eb7e79f8c63c5eaeafa0223580ad63c251ea6a44b0af2458bd960" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest
+    - six
```

Comment:
Are all of these needed at build time? We probably only need `python` and `setuptools`.